Tests/CrossroadTests/OpenURLOptionTests.swift

```diff
@@ -1,9 +1,7 @@
 #if os(iOS)
-
-import Foundation
-import XCTest
 import UIKit
-import Crossroad
+import XCTest
+@testable import Crossroad
```

Comment:
Could you drop `@testable`?

I want to test whether these classes are`public` .

Reply:
No problem, I've changed it to remove `@testable`. There were a few spots where it is required though since the code is using code marked with `internal` so I had to leave those test files with it included.